config/database.yml

```diff
@@ -6,16 +6,14 @@
 default: &default
   adapter: postgresql
   encoding: unicode
-  host: <%= ENV.fetch('APPS_DHH_DEV_DB') %>
-  username: <%= ENV.fetch('APPS_DHH_DEV_DB_USER') %>
-  password: <%= ENV.fetch('APPS_DHH_DEV_DB_PASS') %>
   pool: 5
 variables:
   statement_timeout: 5000
 
 development:
   <<: *default
   database: damspas_development
+  host: <%= ENV.fetch('APPS_DHH_DEV_DB') %>
```

Comment:
Do we still want a default value like 'dams' here? Just curios.

Reply:
Yeah, you know we definitely could. Each docker-compose file and the circleci config set these values. So, in theory, we shouldn't be in the situation where this value isn't set in the `development` or `test` environment context. That said, we could default to `localhost`, which seems kinda safe?

Reply:
Yep, it sounds good to me.

Reply:
Ok I'll do that then, thanks.